### DISCUSSION TITLE
interested in how page search is working use Algolia in the luckyframework.org?

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1914] [Q&A] [answered]
Hi, I want to use create a doc site use lucky, and reference at https://luckyframework.org for how to do it.

I saw we add a link tag in the header, and import and config docsearch in the `src/js/app.js`.

But, i guess there need some config in the `dashboard.algolia.com` for where and how our website to scrap, right?

I create a free plan use my github, when i try to create a crawler, i get following error message.

![image](https://github.com/user-attachments/assets/16bdb163-82db-43ac-b641-917d973261af)

Is there something wrong with my process, or docSearch not available for the free plan?

What I'm more concerned about is whether there are better local free alternatives to using Lucky for the Doc website? (assume doc not saved into pg)

Thanks.

**Comment by jwoertink** [comment, marked as answer]
They've changed stuff since we first signed up, but basically you have to be registered as an open source project. I forget exactly where it is, but there's a site you go to and submit your github URL and if they approve you, then it gets added in.
